doc/lspconfig.txt

```diff
@@ -11,8 +11,8 @@ TABLE OF CONTENTS                                                *lspconfig-toc*
       6a.  Adding Servers      (|lspconfig-adding-servers|)
       6b.  Index               (|lspconfig-index|)
 7.  Root directories           (|lspconfig-root-directories|)
-      6a. Root composition     (|lspconfig-root-composition|)
-      6b. Single file support  (|lspconfig-single-file-support|)
+      7a. Root composition     (|lspconfig-root-composition|)
```

Comment:
Didn't you rename that section?

It should be `lspconfig-root-detection` or `lspconfig-root-dir`, and 7a. should be `lspconfig-root-advanced` (for the tag as well).

Reply:
I think its fixed now